AUTHORS

```diff
@@ -29,6 +29,10 @@ Code from other projects:
   copyright: Dan Haim <negativeiq@users.sourceforge.net>
   license: BSD
 
+* files src/webui/www/private/css/vanillaSelectBox.css src/webui/www/private/scripts/vanillaSelectBox.js
```

Comment:
If those files are are taken from upstream as is (not modified), they should be put into `lib` folder to indicate they are 3rd-party libraries. And we can freely update them to newer versions later.
src/webui/www/private/css/lib/vanillaSelectBox.css
src/webui/www/private/scripts/lib/vanillaSelectBox.js